package.json

```diff
@@ -46,9 +46,5 @@
     "mocha-multi-reporters": "1.5.1",
     "semantic-release": "22.0.7",
     "semantic-release-slack-bot": "4.0.2"
-  },
-  "dependencies": {
-    "@bytecodealliance/wizer": "^3.0.1",
-    "@rollup/rollup-linux-x64-gnu": "^4.4.1"
```

Comment:
These two lines were added by a bot in https://github.com/adobe/helix-rum-collector/commit/c033dd8506de8666eee00053b86f47f4696a7d9e which conveniently has [skip ci] but they break the ci build actually, so I removed them again